portality/static/doaj/js/available_facetviews/public.journalarticle.facetview.js

```diff
@@ -473,7 +474,7 @@ jQuery(document).ready(function($) {
 
         // extract the fulltext link if there is one
         var ftl = false;
-        if (resultobj.bibjson && resultobj.bibjson.link) {
+        if (resultobj.bibjson && resultobj.bibjson.link.length != 0) {
```

Comment:
Hey, sorry, late comment.  I was just porting this over to the new UI code and I spotted a couple of minor issues.  I think this line should actually read:

```
if (resultobj.bibjson && resultobj.bibjson.link && resultobj.bibjson.link.length !== 0) {
```

So, check for `resultobj.bibjson.link` first before looking for the `.length` property, and then the comparison should be `!==` rather than `!=` (due to JS weirdness with the `!=` operator)